README.md

```diff
@@ -1,2 +1,37 @@
 # js-assertion-cheat-sheet
+
 This repository contains examples of various, often used, and tricky assertions in the common JS assertion tools.
+
+## CI/CD
+
+### Feature Branch deployment
+
+> [!IMPORTANT]  
+> Before deploying any feature branches, make sure that the AWS CDK Stack is already deployed.
+
+#### Deploying a Feature Branch
+
+To deploy a feature branch, manually execute the [Feature Branch Deployment](https://github.com/Idea-Pool/js-assertion-cheat-sheet/actions/workflows/feature-branch-deployment.yaml) workflow.
```

Comment:
Did we consider automatic deployment?

Reply:
As I remember, we decided to go with a manual trigger to reduce the number of deployments (assuming that we do not necessarily want to redeploy the feature branch after each push).